terraform/us-dev.tfvars

```diff
@@ -0,0 +1,254 @@
+region = "us-west-2"
+tags   = {}
+env    = "non-prod"
+locals {
+  subnets = [
+    "subnet-0977bacd712bb451a",
+    "subnet-060e02de25cdc9c54",
+    "subnet-0c8462d6423307dca"
+  ]
+  security_groups = [
+    "sg-0a69ed71d8fd50e3b"
+  ]
+  log_group = "/aws/msk-connect/msk-couchbase-events"
+  bootstrap_servers = [
+    "b-1.couchbasebucketstreams.uo4au4.c5.kafka.us-west-2.amazonaws.com:9092",
+    "b-2.couchbasebucketstreams.uo4au4.c5.kafka.us-west-2.amazonaws.com:9092",
```

Comment:
Subnets, security_groups, log_group, bootstrap_servers, and iam_role all need to be passed in via variables and created using their own terraform.